chips/rp2040/src/clocks.rs

```diff
@@ -0,0 +1,1399 @@
+        /// Indicates which src is currently selected (one-hot)
+        (0x068 => clk_adc_selected: ReadOnly<u32, CLK_ADC_SELECTED::Register>),
+        /// Clock control, can be changed on-the-fly (except for auxsrc)
+        (0x06C => clk_rtc_ctrl: ReadWrite<u32, CLK_RTC_CTRL::Register>),
+        /// Clock divisor, can be changed on-the-fly
+        (0x070 => clk_rtc_div: ReadWrite<u32, CLK_RTC_DIV::Register>),
+        /// Indicates which src is currently selected (one-hot)
+        (0x074 => clk_rtc_selected: ReadOnly<u32, CLK_RTC_SELECTED::Register>),
+
+        (0x078 => clk_sys_resus_ctrl: ReadWrite<u32, CLK_SYS_RESUS_CTRL::Register>),
+
+        (0x07C => clk_sys_resus_status: ReadWrite<u32>),
+        /// Reference clock frequency in kHz
+        (0x080 => fc0_ref_khz: ReadWrite<u32>),
+        /// Minimum pass frequency in kHz. This is optional. Set to 0 if you are not using t
+        (0x084 => fc0_min_khz: ReadWrite<u32>),
+        /// Maximum pass frequency in kHz. This is optional. Set to 0x1ffffff if you are not
+        (0x088 => fc0_max_khz: ReadWrite<u32>),
+        /// Delays the start of frequency counting to allow the mux to settle\n
+        /// Delay is measured in multiples of the reference clock period
+        (0x08C => fc0_delay: ReadWrite<u32>),
+        /// The test interval is 0.98us * 2**interval, but let's call it 1us * 2**interval\n
+        /// The default gives a test interval of 250us
+        (0x090 => fc0_interval: ReadWrite<u32>),
+        /// Clock sent to frequency counter, set to 0 when not required\n
+        /// Writing to this register initiates the frequency count
+        (0x094 => fc0_src: ReadWrite<u32>),
+        /// Frequency counter status
+        (0x098 => fc0_status: ReadWrite<u32, FC0_STATUS::Register>),
+        /// Result of frequency measurement, only valid when status_done=1
+        (0x09C => fc0_result: ReadWrite<u32, FC0_RESULT::Register>),
+        /// enable clock in wake mode
+        (0x0A0 => wake_en0: ReadWrite<u32, WAKE_EN0::Register>),
+        /// enable clock in wake mode
+        (0x0A4 => wake_en1: ReadWrite<u32, WAKE_EN1::Register>),
+        /// enable clock in sleep mode
+        (0x0A8 => sleep_en0: ReadWrite<u32, SLEEP_EN0::Register>),
+        /// enable clock in sleep mode
+        (0x0AC => sleep_en1: ReadWrite<u32, SLEEP_EN1::Register>),
+        /// indicates the state of the clock enable
+        (0x0B0 => enabled0: ReadWrite<u32, ENABLED0::Register>),
+        /// indicates the state of the clock enable
+        (0x0B4 => enabled1: ReadWrite<u32, ENABLED1::Register>),
+        /// Raw Interrupts
+        (0x0B8 => intr: ReadWrite<u32>),
+        /// Interrupt Enable
+        (0x0BC => inte: ReadWrite<u32>),
+        /// Interrupt Force
+        (0x0C0 => intf: ReadWrite<u32>),
+        /// Interrupt status after masking & forcing
+        (0x0C4 => ints: ReadWrite<u32>),
+        (0x0C8 => @END),
+    },
+    PllRegisters {
+        /// Control and Status\n
```

Comment:
Nit: Can you remove the line feed control sequences in these comments? Applies throughout the file.

Reply:
done